src/ClientModel/SamplingDataPostProcessor.cpp

```diff
@@ -243,12 +324,20 @@ void SamplingDataPostProcessor::FillThreadSampleDataSampleReports(const CaptureD
 
       function.exclusive = 0;
       function.exclusive_percent = 0.f;
-      auto it = thread_sample_data->resolved_address_to_exclusive_count.find(absolute_address);
-      if (it != thread_sample_data->resolved_address_to_exclusive_count.end()) {
+
+      if (auto it = thread_sample_data->resolved_address_to_exclusive_count.find(absolute_address);
```

Comment:
Also here and below, I would use `contains`.

Reply:
But `it` is used.